README.md

```diff
@@ -101,16 +88,16 @@ The following hardhat commands will achieve this:
 ```shell script
 npx hardhat node --fork https://eth-rinkeby.alchemyapi.io/v2/<<alchemy key>>
 ```
-Alchemy is recommended by Hardhat over Infura because its free accounts provide archived data, which is required for successful forking.
 
-You can then deploy from a separate terminal using the command
+*Note that*: Alchemy is recommended by Hardhat over Infura because its free accounts provide archived data, which is required for successful forking.
+
+You can then deploy from a separate terminal using the command:
 
 ```shell script
 npx hardhat deploy --network localhost
 ```
 
-This makes the BOSON test token deployed on Rinkeby (0xEDa08eF1c6ff51Ca7Fd681295797102c1B84606c) and the official DAI token 
-deployed on Rinkeby (0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D) available to your local hardhat chain.
+This makes the BOSON test token deployed on Rinkeby (0xEDa08eF1c6ff51Ca7Fd681295797102c1B84606c) and the official DAI token deployed on Rinkeby (0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D) available to your local hardhat chain.
```

Comment:
Doe we still want to refer to our Rinkeby deployments, or should we refer to our Ropsten deployments?

Reply:
I left them in here, and then thought we could update the docs when we deploy 1.0.0, and then we can point to instances on all three of the networks.

Reply:
OK